railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_2.rb.tt

```diff
@@ -8,3 +8,16 @@
 #
 # Read the Guide for Upgrading Ruby on Rails for more info on each option.
 # https://guides.rubyonrails.org/upgrading_ruby_on_rails.html
+
+# Controls whether Active Record will automatically look for an inverse relations
+# with a pluralized name.
+#
+# Example:
+#.  class Comment < ActiveRecord::Base
+#     belongs_to :post
+#   end
+#
+# In the above case Active Record will now try to infer a `:comments` relation
+# on `Post`, while previously it would only look for `:comment`.
+#
+# Rails.application.config.active_record.automatic_scope_inversing = :plurals
```

Comment:
Small nitpick, wdyt about `:both_direction` instead of `:plurals` ? `plurals` might be a bit cryptic?